examples/gameroom/gameroom-app/src/store/api.ts

```diff
@@ -144,43 +175,22 @@ export async function listGames(circuitID: string): Promise<Game[]> {
 
 // Payloads
 export async function submitPayload(payload: Uint8Array): Promise<void> {
-  const options = {
-    method: 'POST',
-    url: `http://${window.location.host}/api/submit`,
-    body: payload,
-    headers: {
-      'Content-Type': 'application/octet-stream',
-    },
-  };
-
-  await rp(options).then((body) => {
-    return;
-  })
-  .catch((err) => {
-    console.error(err.message);
-    throw new Error('Failed to send request. Contact administrator for help.');
+  await http('POST', '/submit', payload, (request: XMLHttpRequest) => {
```

Comment:
Do you need to parse the JSON Response on this one?

Reply:
If anything goes wrong at this level, I don't think it should get displayed to the user. I am console.erroring the error in `http()` for debugging.